src/components/inputs/multi-select/multi-select.component.tsx

```diff
@@ -92,44 +121,67 @@ export const MultiSelect: React.FC<FormFieldProps> = ({ question, onChange, hand
-              readOnly={question.readonly}
-            />
+            {question.questionOptions.answers?.length < 5 ? (
+              <CheckboxGroup legendText={question.label} name={question.id}>
+                {question.questionOptions.answers.map((q, index) => {
+                  return (
+                    <Checkbox
+                      key={q.concept}
+                      className={styles.checkbox}
+                      labelText={q.label}
+                      id={q.concept}
+                      onChange={() => {
+                        handleSelectCheckbox(q);
+                        setTouched(true);
+                      }}
+                      name={q.concept}
+                      defaultChecked={initiallyCheckedQuestionItems.some((item) => item === q.concept)}
+                      checked={initiallyCheckedQuestionItems.some((item) => item === q.concept)}
+                      onBlur={onblur}
+                    />
+                  );
+                })}
+              </CheckboxGroup>
+            ) : (
+              <FilterableMultiSelect
+                placeholder={t('search', 'Search') + '...'}
+                onChange={handleSelectItemsChange}
+                id={t(question.label)}
+                items={question.questionOptions.answers
+                  .filter((answer) => !answer.isHidden)
+                  .map((answer, index) => ({
+                    id: `${question.id}-${answer.concept}`,
+                    concept: answer.concept,
+                    label: answer.label,
+                    key: index,
+                    disabled: answer.disable?.isDisabled,
+                  }))}
+                initialSelectedItems={initiallySelectedQuestionItems}
+                label={''}
+                titleText={
+                  question.required ? (
+                    <RequiredFieldLabel label={t(question.label)} />
+                  ) : (
+                    <span>{t(question.label)}</span>
+                  )
+                }
+                key={counter}
+                itemToString={(item) => (item ? item.label : ' ')}
+                disabled={question.disabled}
+                invalid={isFieldRequiredError && errors.length > 0}
+                invalidText={errors[0]?.message}
+                warn={warnings.length > 0}
+                warnText={warnings[0]?.message}
+                readOnly={question.readonly}
+              />
+            )}
           </Layer>
         </div>
         <div className={styles.selectionDisplay}>
-          {field.value?.length ? (
-            <div className={styles.tagContainer}>
-              {handler?.getDisplayValue(question, field.value)?.map((displayValue, index) => (
-                <Tag key={index} type="cool-gray">
-                  {displayValue}
-                </Tag>
-              ))}
-            </div>
+          {field.value?.length && question.questionOptions.answers?.length > 5 ? (
```

Comment:
I would do something like 
```ts
{field.value?.length ? question.questionOptions.answers?.length > 5 ? (
  //display logic for the CheckboxGroup
) : ( 
  //display logic for the `FilterableMulitSelect`
) : (
  <ValueEmpty />
)}
```
That way we don't depreceate the existing logic that was available for displaying selected values for `FilterableMultiSelect`